src/Illuminate/Console/Concerns/ConfiguresPrompts.php

```diff
@@ -238,4 +213,48 @@ protected function restorePrompts()
     {
         Prompt::setOutput($this->output);
     }
+
+    /**
+     * Select fallback with the same behaviour as Prompts.
+     *
+     * @param  string  $label
+     * @param  array  $options
+     * @param  string|int|null  $default
+     * @return string|int
+     */
+    private function selectFallback($label, $options, $default = null)
```

Comment:
I made this method private because it ends up in the `Command` class, and I wanted to prevent IDEs from suggesting it.